source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.tsx

```diff
@@ -276,11 +295,11 @@ class Step1ConfigurationDialog extends Component<Props> {
     const checkboxAcceptance1Field = form.$('checkboxAcceptance1');
     const checkboxAcceptance2Field = form.$('checkboxAcceptance2');
     const walletId = get(wallet, 'id', null);
-    const validRecoveryPhase = recoveryPhraseField.isValid;
-    const walletsDropdownClasses = classnames([
-      styles.walletsDropdown,
-      !validRecoveryPhase ? styles.disabled : null,
-    ]);
+    const walletsDropdownDisabled = !(
+      recoveryPhraseField.isValid ||
+      this.state.wasRecoveryPhraseValidAtLeastOnce
+    );
```

Comment:
```suggestion
    const walletsDropdownDisabled = !recoveryPhraseField.isValid && !this.state.wasRecoveryPhraseValidAtLeastOnce;
```

Reply:
I prefer single negation actually, so first you read `recoveryPhraseField.isValid ||      this.state.wasRecoveryPhraseValidAtLeastOnce` and figure out that these are conditions under which the dropdown should be **enabled**, and later we just negate the whole expression to turn it into **disabled**

Reply:
For me the other way around is simpler to read and understand the expected values of each "ingredient" if the negation happens for each of them. When negating once at the root I need to deduct expected value of each variable. This is however a subjective thing and when I started to search the internet for any good practice regarding this subject I found articles against my suggestion, so I cannot convince you :)

However I have another suggestion that I think both of us will prefer:
```suggestion
    const walletsDropdownEnabled = recoveryPhraseField.isValid ||
      this.state.wasRecoveryPhraseValidAtLeastOnce;
```
What do you think? :)